src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -3547,7 +3547,9 @@ void InstanceKlass::print_on(outputStream* st) const {
     }
   }
   st->print(BULLET"method ordering:   "); method_ordering()->print_value_on(st);      st->cr();
-  st->print(BULLET"default_methods:   "); default_methods()->print_value_on(st);      st->cr();
+  st->print(BULLET"default_methods:   ");
+  if (default_methods() != nullptr) { default_methods()->print_value_on(st); }
+  st->cr();
```

Comment:
The `default_vtable_indicies() printing looks like this:
```
  if (default_vtable_indices() != nullptr) {
    st->print(BULLET"default vtable indices:   "); default_vtable_indices()->print_value_on(st);       st->cr();
  }
```
Should this change make the code follow the same pattern?